### ISSUE TITLE
flag issue to binderhub about ucla dvn not working with mybinder

### ISSUE THREAD

**Issue by jt14den** [ucla-data-archive/ucla-dataverse#21]
add issue to https://github.com/jupyterhub/binderhub that the binder build from dataverse doi not working for us. See the PR: https://github.com/jupyterhub/binderhub/pull/969

Currently when we plug a DOI into mybinder with Dataverse DOI selected, the build holds at "wait" and never moves forward.
 
Curiously, Zenodo will build container, but the share URL will not work.

**Comment by jt14den** [comment]
@jmjamison I haven't tested this in awhile. Can you see if it still isn't working?

**Comment by jmjamison** [comment]
Tested this morning and it's still stuck at 'waiting to build'.  
I've put in an issue:   https://github.com/jupyterhub/binderhub/issues/1133

**Comment by jmjamison** [comment]
Re: jupyterhub issue #1133, the response was that it was a dataverse issue not a jupyter/binder issue.  Dataverse support response was that it should work.  
I went back to test a SSDA DOI on Harvard dataverse and that worked.  Then I tried a recent UCLA dataverse DOI (https://doi.org/10.25346/S6/DE95RT), more than once. The first time was quite slow but subsequent run used the now existing image in the docker repo.
In conclusion I don't know why it started working.  There could have been changes that I'm not aware of.  For now I"m going to close this issue as hypothetically fixed.
